### PR TITLE
Speedup ssr by a s***ton

### PR DESCRIPTION
**Pull request by odinuge**
Stop waterfall fetching stuff in ssr. This cleans up a bit, but require
the fixes in @webkom/react-prepare to properly speedup stuff...

My simple testing gives when running ssr locally (compared to abakus.no):
- `/` (authenticated as me) 0.80s - 1.20s vs 3-4-5sec (and constant timeouts, since 4sec is ssr timeout).

This also fixes some crazy double fetching of quotes...

(might be some rough edges, so I guess we can try it on staging first...)

### CODE REVIEW

**Comment by odinuge** [comment]
Changes in `@webkom/react-prepare` here: https://github.com/webkom/react-prepare/pull/6

File: app/routes/app/AppRoute.js [hunk 1]
```diff
@@ -182,33 +182,33 @@ const mapDispatchToProps = {
...
-      return dispatch(
-        fetchMeetings({
-          dateAfter: moment().format('YYYY-MM-DD'),
-        })
-      );
-    })
+  prepare(
+    (props, dispatch) =>
+      Promise.all([
+        fetchNotificationData(),
```
Comment (SmithPeder) [inline app/routes/app/AppRoute.js:198]: I don't understand everything here, but why does this no longer need a `dispatch`?

Comment (odinuge) [reply app/routes/app/AppRoute.js:198]: ops, good catch!!! :100:

**Review by LudvigHz — APPROVED** [review]
lgtm

**Pull request merged**